src/server/scripts/DragonIsles/zone_the_forbidden_reach.cpp

```diff
@@ -151,6 +153,23 @@ class spell_dracthyr_summon_dervishian : public SpellScript
     }
 };
 
+// 366636 - Stasis
+class spell_stasis_3 : public SpellScript
+{
+    void AcceptQuest()
```

Comment:
It's best to use Load override to check if the caster is a player, so that the spell doesn't load at all. This will prevent sObjectMgr from being called in AfterHit in case GetHitUnit() is not a player. Either that or checking if the target is the player previous to obtaining the questTemplate.

Reply:
I'm not sure what you mean.